plugins/luxtronic2/README.md

```diff
@@ -46,6 +56,12 @@ Defines a mapping to a attribute (read-only). All attribute values are bytes (nu
 ### lux2_c
 Defines a mapping to a calculated value (read-only). All calculated values are integer (numbers).
 
+### lux2_unpack
+Python lambda function. Called before the value is sent towards the device.
+
```

Comment:
The unpack option sounds like the common `eval` setting for items. It evaluates the value and puts the result into the item. IMHO, I prefer to use common settings instead of having special plugin settings for all the plugins.